### PR TITLE
App setup (pt2 - component organisation)

### PR DESCRIPTION
**Pull request by shaunanoordin**
## PR Overview
* Merge #1 first before reviewing this.
* This PR experiments with a way to organise the code - arranging components in a way that, hopefully, make sense for devs.

### CODE REVIEW

**Comment by shaunanoordin** [comment]
## TODO
[ ] Add [Zooniverse-React-Components](https://github.com/zooniverse/Zooniverse-React-Components) to our package.json and use the shared Zooniverse header
[ ] Review `zoo-reduxify`'s recent updates - zooniverse/zoo-reduxify#8 zooniverse/zoo-reduxify#9 zooniverse/zoo-reduxify#10 - and see how the redux components were rearranged into **ducks.** This may need to be another PR though, to keep our changes readable.
[x] Provide a screenshot of how Pt 2 should look like, at this stage of commits:
screen shot 2017-05-19 at 16 27 24

File: src/styles/components/app-header.styl [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    h1  //To be rendered invisible, or removed altogether
+      display: inline-block
+      font-size: 1em
+      vertical-align: middle
+
+  nav
+    flex: 1 1 auto
+    display: flex
+    justify-content: flex-end
+    align-items: center
```
Comment (wgranger) [inline src/styles/components/app-header.styl]: You might need some sort of wrap around here, because the navbar looks quite awkward in mobile view.

**Comment by wgranger** [comment]
For the most part, this looks good, but I made a line comment about mobile views.

Also, concerning the navbar, when clicking on "powered by," the scroll bar that appears pushes the navbar a bit to the left. I'm using Chrome.

**Comment by srallen** [comment]
Re: the components repo. I will be opening a PR soon to add a couple more shared components between PFE and PFE-lab app and also will be fixing the name (so it conforms to NPM convention correctly with lower case). Could y'all wait on adding it until I get that merged in? Also do open issues/PRs for any components you want added and also keep zooniverse/rfc#14 in mind too if we go that route.

**Comment by shaunanoordin** [comment]
@wgranger noted on the mobile view - I haven't even checked how this looks like on mobile!

This may be a good time to query Becky if she has any direction she wants to go to with the mobile design. (Why can't I `@` her here? I'll ping her on Slack)

@srallen Awesome, thanks for the heads up! We can wait for that new PR before we do anything with the shared compos.

**Comment by shaunanoordin** [comment]
I also need to figure out a better way to break my PRs into different stages. I mentioned earlier that PR #1 should be merged first before reviewing this PR, because I thought it'd make reviewing the changes easier on Github.

Unfortunately, looking at the commits in this PR, this PR doesn't seem to recognise commits `750789d` to `853e218` have already been merged into master, and are showing those code changes when looking at 'Files changed'.

Any idea on how to make this PR cleaner asides from me just rebasing this PR on master and then force-pushing the branch?

**Comment by wgranger** [comment]
@shaunanoordin My assumption would be resorting to the good old rebase/force-with-lease, so I'm not sure of an alternative. What would be the argument against that approach here?

**Comment by srallen** [comment]
I've updated Zooniverse-React-Components, so feel free to use it now. It's not on npm, so install via the github repo using the tagged version. It may be getting the talk paginator added to it sometime this week.

**Pull request merged**